golang/cmd/factoryinsight/v2/services/service-tags.go

```diff
@@ -471,109 +470,112 @@ func ProcessCustomTagRequest(c *gin.Context, request models.GetTagsDataRequest)
 	zap.S().Debug("from: ", from)
 	zap.S().Debug("to: ", to)
 
-	var gapFillingMethod string
-
-	switch gapFilling {
-	case models.NoGapFilling:
-		gapFillingMethod = "%s"
-	case models.InterpolationGapFilling:
-		gapFillingMethod = "interpolate(%s)"
-	case models.LocfGapFilling:
-		gapFillingMethod = "locf(%s)"
-	default:
-		helpers.HandleInvalidInputError(
-			c,
-			fmt.Errorf(
-				"invalid gap filling method: %s. Valid values: %s, %s, %s",
-				gapFilling,
-				models.NoGapFilling,
-				models.InterpolationGapFilling,
-				models.LocfGapFilling))
-		return
-	}
-
-	tagAggregates := strings.Split(strings.ReplaceAll(getCustomTagDataRequest.TagAggregates, " ", ""), ",")
-	if len(tagAggregates) == 0 {
-		helpers.HandleInvalidInputError(c, errors.New("invalid tag aggregates"))
-		return
-	}
+	var sqlStatement string
+	if timeBucket == "none" {
 
-	uniqueTagAggregates := make(map[string]bool)
-	for _, tagAggregate := range tagAggregates {
-		uniqueTagAggregates[tagAggregate] = true
-	}
-	tagAggregates = []string{}
-	for tagAggregate := range uniqueTagAggregates {
-		tagAggregates = append(tagAggregates, tagAggregate)
-	}
+		JSONColumnName := enterpriseName + "-" + siteName + "-" + areaName + "-" + productionLineName + "-" + workCellName + "-" + tagName + "-values"
+		data.ColumnNames = []string{"timestamp", JSONColumnName}
 
-	sort.Strings(tagAggregates)
+		// #nosec G201
+		sqlStatement = `
+SELECT
+	asset_id,
+	timestamp,
+	value
+FROM
+	processvaluetable
+WHERE
+    asset_id = $1 AND
+    valuename = $2 AND
+    timestamp >= $3 AND
+    timestamp <= $4
+GROUP BY asset_id, timestamp, value
```

Comment:
Pls add sorting by timestamp, to fix this:
![image](https://user-images.githubusercontent.com/6317420/199990680-7cd752ec-874b-43c1-bb00-506c2a9f3de9.png)